client/landing/jetpack-cloud/index.js

```diff
@@ -1,25 +1,84 @@
 /**
  * External dependencies
  */
-import config from '../../config';
+import page from 'page';
 
 /**
  * Internal dependencies
  */
-import initJetpackCloudRoutes from './routes';
-import { bootApp } from 'boot/common';
 
-/**
- * Style dependencies
- */
-import 'components/environment-badge/style.scss';
-import 'layout/style.scss';
-import 'assets/stylesheets/jetpack-cloud.scss';
-
-window.AppBoot = () => {
-	if ( ! config.isEnabled( 'jetpack-cloud' ) ) {
-		window.location.href = '/';
-	} else {
-		bootApp( 'Jetpack Cloud', initJetpackCloudRoutes );
-	}
-};
+import { navigation, siteSelection, sites } from 'my-sites/controller';
+import { makeLayout, render as clientRender } from 'controller';
+
+import { dashboard } from './sections/dashboard/controller';
+import {
+	backups,
+	backupDetail,
+	backupDownload,
+	backupRestore,
+} from './sections/backups/controller';
+import { scan, scanHistory } from './sections/scan/controller';
+import { settings } from './sections/settings/controller';
+
+export default function() {
+	page( '/', siteSelection, navigation, dashboard, makeLayout, clientRender );
```

Comment:
Is `siteSelection` really necessary for this root route? Will we be redirecting to `/:siteSlug` when the logged in user lands?

Reply:
Right now I don't have a good idea for what the dashboard holds. I think this can be addressed in a seperate PR when we revisit the dashboard or the lack of it.